src/blob_file_cache.cc

```diff
@@ -66,11 +65,16 @@ Status BlobFileCache::FindFile(uint64_t file_number, uint64_t file_size,
     return s;
   }
   std::unique_ptr<RandomAccessFileReader> file;
+  uint64_t file_size;
   {
     std::unique_ptr<FSRandomAccessFile> f;
     auto file_name = BlobFileName(db_options_.dirname, file_number);
-    s = env_->GetFileSystem()->NewRandomAccessFile(
-        file_name, FileOptions(env_options_), &f, nullptr /*dbg*/);
+    auto fs = env_->GetFileSystem();
+
+    s = fs->GetFileSize(file_name, IOOptions(), &file_size, nullptr);
```

Comment:
Why do we bother to get the file size from again (which is a bit less efficient than passing in parameter) , when the current code already has the size in the context.